Week6/sheepyis/Mission2/src/components/lp/detail/comment.tsx

```diff
@@ -0,0 +1,100 @@
+import { useState, useEffect } from "react";
+import { useLpCommentInfiniteQuery } from "../../../hooks/lp/useLpCommentInfiniteQuery";
+import { useInView } from "react-intersection-observer";
+import ListComment from "./list-comment";
+import SkeletonComment from "../../skeleton/skeletonComment";
+
+const Comment = () => {
+  const [order, setOrder] = useState<"asc" | "desc">("desc");
+
+  const {
+    data,
+    fetchNextPage,
+    hasNextPage,
+    isFetchingNextPage
+  } = useLpCommentInfiniteQuery(1, order, 5);
+
+  const { ref, inView } = useInView({ threshold: 1 });
```

Comment:
현재 코드에서 `threshold: 1`로 설정되어 있어 해당 요소가 100% 뷰포트에 들어와야 inView가 true로 변경됩니다!
`threshold` 값을 낮춰서 요소가 일부만 보여도 감지되게 하면 기존에 발생하던 스크롤 감지 오류가 개선됩니다!!
추가적으로, `rootMargin` 옵션을 통해 더 자연스럽게 수정할 수 있을 것 같아요!
(참고 : https://www.heropy.dev/p/ydKoQO)

```suggestion
  const { ref, inView } = useInView({ threshold: 0.1 });
```